File: Zee/Form2.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using static iNovation.Code.General;
using static iNovation.Code.Desktop;
namespace Zee
{
    public partial class Form2 : Form
    {
        public Form2()
        {
            InitializeComponent();
        }
        private enum Profession 
        {
            Doctor, Pilot, Engineer, Writer, Hairdresser
        }

        private void Form2_Load(object sender, EventArgs e)
        {
            TitleDrop(titleDropDown);
            GenderDrop(genderDropDown, false);
            BindProperty(professionDropDown, GetEnum(new Profession()), false);

        }

        private void resetButton_Click(object sender, EventArgs e)
        {
            Control[] controls = {titleDropDown, nameTextBox, genderDropDown, professionDropDown, yearsOfExperienceTextBox };            
            Clear(controls);
        }

        private void submitButton_Click(object sender, EventArgs e)
        {
            summaryTextBox.Text = Content(titleDropDown) + Environment.NewLine + Content(nameTextBox) + Environment.NewLine + Content(genderDropDown) + Environment.NewLine + Content(professionDropDown) + Environment.NewLine + Content(yearsOfExperienceTextBox);
        }

        private void copyToListBoxButton_Click(object sender, EventArgs e)
        {
            BindProperty(summaryListBox, StringToList(Content(summaryTextBox)));
        }

        private void upButton_Click(object sender, EventArgs e)
        {
            mFeedback("working", "this appears to be working");
        }

        private void removeButton_Click(object sender, EventArgs e)
        {
            ListsRemoveItem(summaryListBox);
        }

        private void clearButton_Click(object sender, EventArgs e)
        {
            ListsClearList(summaryListBox);
        }
    }
}

File: Scratch/Form2.vb
Imports iNovation.Code.General
Imports iNovation.Code.Desktop
Imports iNovation.Code.Machine
Imports Microsoft.Win32
Imports iNovation.Code.Media
Imports System.Text.RegularExpressions
Imports iNovation.Code ''.CheckedDifference
Imports iNovation.Code.SearchEngineQueryString
Imports System.Collections.ObjectModel
Public Class Form2

    Private Enum Profession
        Doctor
        Pilot
        Engineer
        Writer
        Hairdresser
    End Enum

    Private Sub Form2_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        TitleDrop(TitleDropDown)
        GenderDrop(GenderDropDown, False)
        BindProperty(ProfessionDropDown, GetEnum(New Profession), False)
    End Sub

    Private Sub ResetButton_Click(sender As Object, e As EventArgs) Handles ResetButton.Click
        Clear({TitleDropDown, NameTextBox, GenderDropDown, ProfessionDropDown, YearsOfExperienceTextBox})
    End Sub

    Private Sub SubmitButton_Click(sender As Object, e As EventArgs) Handles SubmitButton.Click
        SummaryTextBox.Text = Content(TitleDropDown) & vbCrLf & Content(NameTextBox) & vbCrLf & Content(GenderDropDown) & vbCrLf & Content(ProfessionDropDown) & vbCrLf & Content(YearsOfExperienceTextBox)
    End Sub

    Private Sub CopyToListBoxButton_Click(sender As Object, e As EventArgs) Handles CopyToListBoxButton.Click
        BindProperty(SummaryListBox, StringToList(Content(SummaryTextBox)))
    End Sub

    Private Sub UpButton_Click(sender As Object, e As EventArgs) Handles UpButton.Click
        mFeedback("Working", "this appears to be working")
    End Sub

    Private Sub RemoveButton_Click(sender As Object, e As EventArgs) Handles RemoveButton.Click

    End Sub

    Private Sub ClearButton_Click(sender As Object, e As EventArgs) Handles ClearButton.Click

    End Sub
End Class